CMakeLists.txt

```diff
@@ -47,7 +48,29 @@ if (CMAKE_COMPILER_IS_GNUCC)
 	set (CMAKE_C_FLAGS_RELEASE "-DNDEBUG")
 	set (CMAKE_C_FLAGS_RELWITHDEBINFO "-gsplit-dwarf")
 	add_compile_options(-Wall -Wextra -Wformat -pedantic -std=gnu99 -Wstrict-prototypes -Wsign-compare)
-	add_compile_options(-Wredundant-decls -Wformat-overflow)
+	add_compile_options(-Wredundant-decls)
```

Comment:
correct, I was doing tests and forgot to check if Wformat-overflow was implied by Wall.